### PR TITLE
Change netty version to 4.1.5.Final

### PR DESCRIPTION
**Pull request by paulgallagher75**

### CODE REVIEW

File: artemis-protocols/artemis-mqtt-protocol/pom.xml [hunk 1]
```diff
@@ -47,29 +47,12 @@
...
-         <artifactId>netty-codec-mqtt</artifactId>
+         <artifactId>netty-all</artifactId>
       </dependency>
       <dependency>
          <groupId>junit</groupId>
          <artifactId>junit</artifactId>
       </dependency>
    </dependencies>
 
-   <build>
```
Comment (clebertsuconic) [inline artemis-protocols/artemis-mqtt-protocol/pom.xml]: @mtaylor can you look into this?

Comment (mtaylor) [inline artemis-protocols/artemis-mqtt-protocol/pom.xml]: @paulgallagher75 is going to test the build against MQTT example.  Should be fine though, the codec should be included in the lib.

**Comment by tabish121** [comment]
Netty All doesn't include OSGi bundle info in case that is relevant.

**Comment by clebertsuconic** [comment]
@tabish121 I should have listened to you more Careful. ArtemisFeatureTest is failing because of this. I thought it passed but I looked it wrongly.


It's ok thought.. at least we know where is the failure and how to fix it.


@tabish121 / @paulgallagher75  how to fix it?

**Comment by gemmellr** [comment]
Reverting the replacement of netty-codec-mqtt with netty-all (but keeping the version change) might work, as the individual netty-foo modules do have OSGi metadata, and that would be a closer equivalent to what was being done previously.

**Comment by clebertsuconic** [comment]
@paulgallagher75  can you do that?

**Comment by clebertsuconic** [comment]
@paulgallagher75  it still didn't work actually... I will ask @dejanb 's help :)

**Pull request closed without merging**